custom_typings/polymer-build.d.ts

```diff
@@ -76,6 +76,8 @@ declare module 'polymer-build' {
   }
 
   export interface AddServiceWorkerOptions {
+    project: PolymerProject;
```

Comment:
we should be pulling these typings out of polymer-build. polymer-build needs a `"declaration": true` line in its tsconfig